mora/exceptions.py

```diff
@@ -49,18 +49,18 @@ def get_body(self, environ=None):
 
 
 class ValidationError(BaseError):
-    key = Error.E90000
+    key = ErrorCodes.E_INVALID_INPUT
     cause = 'validation'
     code = 400
 
 
 class NotFoundError(BaseError):
-    key = Error.E90003
+    key = ErrorCodes.E_NOT_FOUND
     cause = 'not-found'
     code = 404
```

Comment:
I'd like us to get rid of these exception, and encode the `code` somewhere else — such as in your prefix character.

Reply:
So you want it to be `E404_NOT_FOUND`?

Reply:
Or perhaps:

```python
E_NOT_FOUND = 404, 'go fetch'
```

Reply:
I guess that makes sense. So we would only have a single error class, essentially, and have the individual codes be defined in `errorcodes`?